ddtrace/contrib/dbapi/__init__.py

```diff
@@ -69,10 +69,13 @@ def __enter__(self):
 class TracedConnection(wrapt.ObjectProxy):
     """ TracedConnection wraps a Connection with tracing code. """
 
-    def __init__(self, conn):
+    def __init__(self, conn, pin=None):
         super(TracedConnection, self).__init__(conn)
         name = _get_vendor(conn)
-        Pin(service=name, app=name).onto(self)
+        if pin and pin.enabled():
+            pin.onto(self)
+        else:
+            Pin(service=name, app=name, app_type="db").onto(self)
```

Comment:
instead of using `db` can you:
```python
from ...ext import http, AppTypes  # from here should be the right import

Pin(service=name, app=name, app_type=AppTypes.db).onto(self)
```

Reply:
It should be applied everywhere since we're here at this point.